src/mongo_logger.ts

```diff
@@ -430,14 +430,18 @@ export class MongoLogger {
   maxDocumentLength: number;
   logDestination: MongoDBLogWritable | Writable;
 
+  error = this.log.bind(this, 'error');
+  warn = this.log.bind(this, 'warn');
+  info = this.log.bind(this, 'info');
+  debug = this.log.bind(this, 'debug');
+  trace = this.log.bind(this, 'trace');
```

Comment:
Can we leave tsdoc about when to use each. For example an important detail about error from the spec is that it is to be used when there is an error that cannot be reported via API methods